src/test/java/org/apache/brooklyn/cloudfoundry/entity/VanillaPaasApplicationCloudFoundryDriverTest.java

```diff
@@ -52,7 +54,6 @@
 import com.squareup.okhttp.mockwebserver.RecordedRequest;
 
 public class VanillaPaasApplicationCloudFoundryDriverTest extends AbstractCloudFoundryUnitTest {
```

Comment:
@andreaturli just couple of questions about this test. 
This test is a unit test, the location was mocked.
Should we add an integration where the location will not be mocked (using `FakeCloudFoundryClient`)?
In case of adding the integration test, should I remove this unit test?